### PR TITLE
GBC-737: log when job enters queue and when it starts

### PR DESCRIPTION
**Pull request by bronter**

### CODE REVIEW

File: lib/resque/plugins/datadog.rb [hunk 1]
```diff
@@ -4,25 +4,35 @@ module Datadog
       mattr_accessor :statsd
       mattr_accessor :service_name
 
+      def after_enqueue_datadog(*args)
+        return unless statsd
+        @enqueued_at = Time.now
+        statsd.event('Job Enqueued', enqueued_at.to_s, tags: job_tags(args))
```
Comment (jonlouie) [inline lib/resque/plugins/datadog.rb]: Should `enqueued_at.to_s` be `@enqueued_at.to_s`?

Comment (bronter) [reply lib/resque/plugins/datadog.rb]: Yep, good catch!

**Review by jonlouie — APPROVED** [review]

**Comment by subimage** [comment]
This change looks good, and probably fundamental to getting those Datadog dashboards built we need. What's the holdup in merging? @bronter @jlouie3

**Comment by subimage** [comment]
I'd also wonder if there's a spec we should be creating to test this plugin? It's great @jlouie3 caught that error, but we should be more confident in this code through automated testing. If I can help writing a spec for this, or instruct how we might go about testing it, please let me know. cc: @f3z0

**Review by f3z0 — CHANGES_REQUESTED** [review]
Can we add some rspec for this before we put it into production

**Comment by f3z0** [comment]
Also I just heard that we are moving to mono-repo, so we can go ahead and move this into rails-api and put it into the gems folder

**Comment by bronter** [comment]
@f3z0 / @subimage Yeah, I actually started writing specs for it but I had to set up `rspec` myself for this repo since we don't have any tests here yet, and getting the mocks set up was taking a lot of time, kinda didn't want it to drag on.  If we're moving it into a mono-repo anyways setting up rspec will be a much more rewarding investment.

**Comment by subimage** [comment]
Gabe, see the work I’ve done in this pull for Menu Feeds <https://github.com/greenbits/herer-api/pull/4052> when it comes to setting up RSpec for an internal monorepo gem. I believe you can build on that progress. Happy to walk you through it also.

// seth banks
// green bits — principal designer

**Comment by bronter** [comment]
Since we're moving this code into herer-api gonna close this PR

**Pull request closed without merging**